.circleci/config.yml

```diff
@@ -304,7 +304,9 @@ jobs:
       - npm-install
       - run:
           name: Build JavaScript Bundle
-          command: npm run bundle:android
+          command: |
+            npm run core prebundle
```

Comment:
We also have to update the localizations when publishing the bundle for Android.